make/Makefile.common

```diff
@@ -91,7 +91,7 @@ ifneq (, $(filter $(compiler), clang clang++))
     include $(proj_root)/make/clang.inc
 endif
 
-ifneq (, $(filter $(compiler), icc icpc))
+ifneq (, $(filter $(compiler), icc icpc icpx))
```

Comment:
As far as I know `icpx` is closer to `dpcpp` than to `icpc`. So `icpx` and `icpc` may have incompatible options.